set.mm

```diff
@@ -594199,11 +594196,51 @@ X C ( ._|_ ` Y ) ) ) $=
     glbconN $p |- ( ( K e. HL /\ S C_ B )
         -> ( G ` S ) = ( ._|_ ` ( U ` { x e. B | ( ._|_ ` x ) e. S } ) ) ) $=
       ( vz vu wcel cfv wceq wbr wral wi wa vy vw vv vt wss chlt cv crab sseqin2
       cin biimpi dfin5 eqtr3di fveq2d cple crio eqid biid id ssrab2 glbval cops
+      a1i wreu hlop ccla cdm hlclat clatglbcl2 syl2anc glbeu breq1 breq2 imbi2d
+      ralbidv anbi12d riotaocN ad2antrr opoccl sylancom wrex opococ rspceeqv wb
+      eqcomd fveq2 eleq1 imbi12d adantl ralxfrd simpr simplr oplecon3b ralbidva
+      syl3anc bitr4d ralrab weq eleq1d 3bitr4g ad3antrrr riotabidva simpl mpan2
+      lubval eqtr4d 3eqtrd sylan9eqr ) CBUEZFUFNZCEOAUGZCNZABUHZEOZXKGOZCNZABUH
+      ZDOZGOZXICXMEXIBCUJZCXMXIXTCPCBUIUKABCULUMUNXJXNUAUGZLUGZFUOOZQZLXMRZUBUG
+      ZYBYCQZLXMRZYFYAYCQZSZUBBRZTZUABUPZUCUGZGOZYBYCQZLXMRZYHYFYOYCQZSZUBBRZTZ
+      UCBUPZGOZXSXJYLUALUBBXMEFYCUFHYCUQZJYLURZXJUSZXMBUEZXJXLABUTVCZVAXJFVBNZY
+      LUABVDYMUUCPFVEZXJYLUALUBBXMEFYCUFHUUDJUUEUUFXJFVFNUUGXMEVGNFVHUUHBXMEFHJ
+      VIVJVKYLUUAUAUCBFGHKYAYOPZYEYQYKYTUUKYDYPLXMYAYOYBYCVLVOUUKYJYSUBBUUKYIYR
+      YHYAYOYFYCVMVNVOVPVQVJXJUUBXRGXJUUBMUGZYNYCQZMXQRZUULUDUGZYCQZMXQRZYNUUOY
+      CQZSZUDBRZTZUCBUPZXRXJUUAUVAUCBXJYNBNZTZYQUUNYTUUTUVDYBCNZYPSZLBRZUULGOZC
+      NZUUMSZMBRZYQUUNUVDUVGUVIYOUVHYCQZSZMBRUVKUVDUVFUVMLMUVHBBUVDUULBNZUUIUVH
+      BNZXJUUIUVCUVNUUJVRZBFGUULHKVSZVTUVDYBBNZTZYBGOZBNZYBUVTGOZPZYBUVHPZMBWAZ
+      UVDUVRUUIUWAXJUUIUVCUVRUUJVRZBFGYBHKVSZVTUVSUWBYBUVDUVRUUIUWBYBPZUWFBFGYB
+      HKWBZVTWEMUVTBUVHUWBYBUULUVTGWFWCZVJUWDUVFUVMWDUVDUWDUVEUVIYPUVLYBUVHCWGZ
+      YBUVHYOYCVMWHWIWJUVDUVJUVMMBUVDUVNTZUUMUVLUVIUWLUUIUVNUVCUUMUVLWDUVPUVDUV
+      NWKXJUVCUVNWLBFYCGUULYNHUUDKWMWOVNWNWPXLUVEYPLABXKYBCWGZWQXPUVIUUMMABAMWR
+      XOUVHCXKUULGWFWSZWQWTUVDYTUUOGOZYBYCQZLXMRZUWOYOYCQZSZUDBRUUTUVDYSUWSUBUD
+      UWOBBUVDUUOBNZUUIUWOBNXJUUIUVCUWTUUJVRZBFGUUOHKVSVTUVDYFBNZTZYFGOZBNZYFUX
+      DGOZPYFUWOPZUDBWAUVDUXBUUIUXEXJUUIUVCUXBUUJVRZBFGYFHKVSVTUXCUXFYFUVDUXBUU
+      IUXFYFPUXHBFGYFHKWBVTWEUDUXDBUWOUXFYFUUOUXDGWFWCVJUXGYSUWSWDUVDUXGYHUWQYR
+      UWRUXGYGUWPLXMYFUWOYBYCVLVOYFUWOYOYCVLWHWIWJUVDUUSUWSUDBUVDUWTTZUUQUWQUUR
+      UWRUXIUVIUUPSZMBRZUVEUWPSZLBRZUUQUWQUXIUXKUVIUWOUVHYCQZSZMBRUXMUXIUXJUXOM
+      BUXIUVNTZUUPUXNUVIUXPUUIUVNUWTUUPUXNWDXJUUIUVCUWTUVNUUJXAZUXIUVNWKUVDUWTU
+      VNWLBFYCGUULUUOHUUDKWMWOVNWNUXIUXLUXOLMUVHBBUXIUVNUUIUVOUXQUVQVTUXIUVRTZU
+      WAUWCUWEUXIUVRUUIUWAXJUUIUVCUWTUVRUUJXAZUWGVTUXRUWBYBUXIUVRUUIUWHUXSUWIVT
+      WEUWJVJUWDUXLUXOWDUXIUWDUVEUVIUWPUXNUWKYBUVHUWOYCVMWHWIWJWPXPUVIUUPMABUWN
+      WQXLUVEUWPLABUWMWQWTUXIUUIUVCUWTUURUWRWDUXAXJUVCUWTWLUVDUWTWKBFYCGYNUUOHU
+      UDKWMWOWHWNWPVPXBXJXQBUEZXRUVBPXPABUTXJUXTTUVAUCMUDBXQDFYCUFHUUDIUVAURXJU
+      XTXCXJUXTWKXEXDXFUNXGXH $.
+
+    $( Obsolete version of ~ glbconN as of 3-Jan-2025.  (Contributed by NM,
+       17-Jan-2012.)  (New usage is discouraged.)
+       (Proof modification is discouraged.) $)
+    glbconNOLD $p |- ( ( K e. HL /\ S C_ B )
```

Comment:
Instead of OLD, we should use the suffix ALT here, so that this theorem remains discouraged, but will not be deleted after a year.

Reply:
since the new proof is shorter, I think that basically functions as an OLD-revision regardless of the axiom removal

Reply:
I think we want that theorem to be ultimately deleted, so we deprecate it and label it *OLD.  This is what was planned in the comment of ~ax-riotaBAD

Reply:
OK, since it seems that there is no use of it anymore, it can remain OLD and be removed after a year.